docs/content/documentation/content/page.md

```diff
@@ -104,4 +104,4 @@ available separately in the
 
 An anchor link to this position named `continue-reading` is created so you can link
 directly to it if needed for example:
-`<a href="{{ page.permalink }}#continue-reading">Continue Reading</a>`
+`<p  id="zola-continue-reading"><a href="{{ page.permalink }}#continue-reading">Continue Reading</a></p>`
```

Comment:
I don't think you need to wrap that one

Reply:
do you mean wrapping the `a` tag with a `p` tag?

the `p` tag is the one with the id since and I wanted to expose the behavior in the docs somewhere.

Reply:
ah mis understood that piece of documentation. corrected it.